test/integration/scheduler_perf/scheduler_perf.go

```diff
@@ -375,14 +380,31 @@ func (p *params) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-// get returns param.
+// getParam retrieves the parameter as an integer
```

Comment:
```suggestion
// get retrieves the parameter as an integer
```

Reply:
Fixed. I had initially experimented with turning this into a generic method, but Go does not support those.